views/config_postProcessing.mako

```diff
@@ -200,6 +200,18 @@
                                     <span class="component-desc"><b>NOTE:</b> Only working with RAR archive</span>
                                 </label>
                             </div>
+                            <div id="content_unpack">
```

Comment:
@p0psicles @medariox so this is wrong? I explained why and know I'm pointing why. If is still wrong need to talk with omg

```javascript
    $('.enabler').each(function() {
        if (!$(this).prop('checked')) {
            $('#content_' + $(this).attr('id')).hide();
        }
    });
```
https://github.com/pymedusa/Medusa/blob/develop/static/js/common/init.js#L247